libbeat/cmd/configtest.go

```diff
@@ -15,6 +16,7 @@ func genConfigTestCmd(name, version string, beatCreator beat.Creator) *cobra.Com
 		Run: func(cmd *cobra.Command, args []string) {
 			b, err := beat.New(name, version)
 			if err != nil {
+				fmt.Fprintf(os.Stderr, "Error initializing beat: %s\n", err)
```

Comment:
I just don't like it that logp and fmt.Println are interleaved in the code. But def. not a show stopper.